airbyte-integrations/connectors/source-prestashop/source_prestashop/schemas/orders.json

```diff
@@ -147,52 +147,50 @@
       "properties": {
         "order_rows": {
           "type": "array",
-          "items": [
-            {
-              "type": "object",
-              "properties": {
-                "id": {
-                  "type": "string"
-                },
-                "product_id": {
-                  "type": "string"
-                },
-                "product_attribute_id": {
-                  "type": "string"
-                },
-                "product_quantity": {
-                  "type": "string"
-                },
-                "product_name": {
-                  "type": "string"
-                },
-                "product_reference": {
-                  "type": "string"
-                },
-                "product_ean13": {
-                  "type": "string"
-                },
-                "product_isbn": {
-                  "type": "string"
-                },
-                "product_upc": {
-                  "type": "string"
-                },
-                "product_price": {
-                  "type": "string"
-                },
-                "id_customization": {
-                  "type": "string"
-                },
-                "unit_price_tax_incl": {
-                  "type": "string"
-                },
-                "unit_price_tax_excl": {
-                  "type": "string"
-                }
+          "items": {
+            "type": "object",
+            "properties": {
+              "id": {
+                "type": "string"
+              },
+              "product_id": {
+                "type": "string"
+              },
+              "product_attribute_id": {
+                "type": "string"
+              },
+              "product_quantity": {
+                "type": "string"
+              },
+              "product_name": {
+                "type": "string"
+              },
+              "product_reference": {
+                "type": "string"
+              },
+              "product_ean13": {
+                "type": "string"
+              },
+              "product_isbn": {
+                "type": "string"
+              },
+              "product_upc": {
+                "type": "string"
+              },
+              "product_price": {
```

Comment:
I'm pretty sure this is a float, is "string" the only data type allowed at the moment?

For the record, I was part of the PrestaShop Core Team for 4 years.

Reply:
Hay @mickaelandrieu 
all types are supported: "string", "number",  "integer", "boolean"
are you talking about "product_price" field only ?

Reply:
@mickaelandrieu can you make PR  for all fields which has incorrect type ?

Reply:
yes I will asap, maybe during the week end :)